easylist/easylist_adservers.txt

```diff
@@ -20530,6 +20544,7 @@
 ||pubmatic.com^$third-party
 ||pubwise.io^$third-party
 ||puffnetwork.com^$third-party
+||putlockertv.com^$third-party
```

Comment:
@felippesantos46 what do you have on this record. If you visit the domain it seems pretty much like a pirated domains